client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/step-content.tsx

```diff
@@ -1,191 +0,0 @@
-/**
```

Comment:
`NB:` We should consider whether or not anything else in the `step-content` component needs testing before removing the scaffolding entirely

Reply:
@jeyip - How about if I add a separate PR for myself to re-add the scaffolding along with some basic tests for this file? I can tackled that this week - maybe not with a complete test suite, but with some updated test scaffolding and basic tests that we can continue to fill out.

There is indeed a lot we can potentially test for step-content. I didn't want to try to tackle that here because I'd need to remove the existing test and write some other tests that aren't directly related to the PR.

Reply:
To confirm - yes, I'm just going to circle back and added more generic, updated tests for this file separately.